Problem1_Hashmap.java

```diff
@@ -0,0 +1,40 @@
+package design1;
+
```

Comment:
You will run out of memory when the number of elements becomes large. You should implement some HASH function and rehash later if load factor crosses threshold.